package.json

```diff
@@ -1,17 +1,20 @@
 {
   "repository": "https://github.com/kiltprotocol/sdk-js",
+  "name": "root-workspace",
   "private": true,
   "workspaces": [
     "packages/*",
     "docs/*"
   ],
   "license": "BSD-4-Clause",
   "scripts": {
-    "check": "tsc -p tsconfig.json --noEmit",
-    "build": "yarn workspaces foreach -p -t --exclude '{root-workspace}' run build",
+    "check": "tsc -b tsconfig.json",
+    "build": "yarn clean && yarn workspaces foreach -pt --exclude 'root-workspace' run build",
+    "build:dev": "yarn workspaces foreach -pt --exclude 'root-workspace' run build:cjs",
```

Comment:
I guess this is to be used before unit/integration tests? Can we mention it somewhere? Otherwise I am 99% sure people will still be running `yarn build`.

Reply:
True, I haven't updated READMEs and such. Could also add comments above the respective scripts

Reply:
Okay so we don't have a section on development in the README and we can't add comments to the package.json, where do you think we should put that info?

Reply:
I guess the README then? I don't see any other place where this could go, personally. Maybe @arty-name has some prior experience for this?